public/js/actions/breakpoints.js

```diff
@@ -31,9 +25,17 @@ function _getOrCreateBreakpoint(state, location, condition) {
   return getBreakpoint(state, location) || fromJS({ location, condition });
 }
 
+function enableBreakpoint(location: Location) {
+  // Enabling is exactly the same as adding. It will use the existing
+  // breakpoint that still stored.
+  return addBreakpoint(location);
+}
+
 function addBreakpoint(location: Location,
                        { condition, getTextForLine } : any = {}) {
   return ({ dispatch, getState, client }: ThunkArgs) => {
+    location = getOriginalLocation(getState(), location);
```

Comment:
Do we need to do this here? Any location coming from the UI should already be an original location, but if you want to do it for sanity, that's fine. Once we actually type locations we should be able to enforce that `location` is an original location and we shouldn't need to do this.